src/tests/class_tests/openms/source/EmpiricalFormula_test.cpp

```diff
@@ -387,15 +387,30 @@ START_SECTION(IsotopeDistribution getIsotopeDistribution(UInt max_depth) const)
     TEST_REAL_SIMILAR(it->getMZ(), result_rounded_mass[i])
   }
 
-  // Formula of Bovine Serum Albumin (BSA) that is has been processed and is missing the first 24 AA
+  // Formula of Bovine Serum Albumin (BSA) that has been processed and is missing the first 24 AA
   EmpiricalFormula ef2("C2934H4615N781O897S39");
-  IsotopeDistribution iso3 = ef2.getIsotopeDistribution(CoarseIsotopePatternGenerator(20));
+  IsotopeDistribution iso3 = ef2.getIsotopeDistribution(CoarseIsotopePatternGenerator(100));
   // monoisotopic mass
   TEST_REAL_SIMILAR(iso3.begin()->getMZ(), 66389.863)
-
-  IsotopeDistribution iso4 = ef2.getIsotopeDistribution(CoarseIsotopePatternGenerator(20, true));
+  // the average mass of the IsotopeDistribution is computed from its masses and probabilities
+  // and should be extremely close to the result from the EmpiricalFormula which uses the
+  // average weights of each atom.
+  TEST_REAL_SIMILAR(iso3.averageMass(), ef2.getAverageWeight())
+
+  Peak1D most_abundant = iso3.getMostAbundant();
+  // Partially a regression test. For such a large molecule, consecutive isotopes have very similar probabilities
+  // and a small change could change the result by 1Da (e.g. numerical error, elemental isotope probabilities)
+  TEST_REAL_SIMILAR(most_abundant.getMZ(), 66432.0037);
```

Comment:
why isnt this `66389.863 + C12_C13 * k` for an integer value of `k` ? Did I misinterpret something here?

Reply:
Your got the equation right: C12_C13 = 1.00335 and in this case k = 42, so 66389.863 + (1.00335 * 42) = 66389.0037.

Related to this, I see that OpenSWATH DIAHelper uses a 1.00048 distance between isotopes. https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h#L63-L67. 

I'm comparing that with using 1.00335 right now. I've never used the tool before but I imagine some of those isotope fields in the output might improve. When I switched FeatureFinderMultiplex from using 1.00335 to 1.0, the number of features went from ~27k to ~25k

Reply:
No effect on OpenSWATH, but I'm not going to dig into why.

Reply:
> No effect on OpenSWATH, but I'm not going to dig into why.

ok, that is fine :-)